coffee_cmd/src/main.rs

```diff
@@ -98,6 +98,17 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Nurse {} => {
+            let mut spinner = term::spinner(&format!("Refreshing storage"));
```

Comment:
```suggestion
            let mut spinner = term::spinner(&format!("Trying restoring from a corrupted status!"));
```

We should also check if there is a corrupted status, if not we return a message that there is no need to run the command

Reply:
I modified the nurse function to return a json object with the status of the file and print a message to the user accordingly.